src/v/storage/segment_appender_chunk.h

```diff
@@ -63,9 +63,19 @@ class segment_appender_chunk {
         // must be 8192 bytes, starting at the bottom of the _flushed_pos
         // page, in this example, at offset 0.
         //
-        const auto prev_sz = ss::align_down<size_t>(_flushed_pos, _alignment);
-        const auto curr_sz = ss::align_up<size_t>(_pos, _alignment);
-        return curr_sz - prev_sz;
+        return pending_aligned_end() - pending_aligned_begin();
+    }
+
+    // The aligned start (inclusive) position within the chunk of the pending
+    // (unflushed) region, i.e., align_down(_pos).
+    size_t pending_aligned_begin() const {
```

Comment:
We need these from outside now. We don't actually use dma_size (above) in the prod code anymore but it still has one caller in tests.